### ISSUE TITLE
[CU-2zhvxrm] Admin response author field and indication are not populated

### ISSUE THREAD

**Issue by cyp3rius** [VirtusLab-Open-Source/strapi-plugin-comments#162]
Hi, I have tried responding direct from the panel (version `2.2.0`) but the `author` object won't get populated completely:
```js
{
  avatar: null,
  email: "lorem@ipsum.com",
  id: "1",
  name: null
}
```

Despite the fact that this user is a super admin and does have first and last name, they both are ignored. What's more, right know there is no indicator that this comment comes from the admin or a staff member.

_Originally posted by @kwiat1990 in https://github.com/VirtusLab-Open-Source/strapi-plugin-comments/issues/55#issuecomment-1280018526_

**Comment by cyp3rius** [comment]
ClickUp tracking: #2zhvxrm

**Comment by cyp3rius** [comment]
@kwiat1990 this has been fixed in `v2.2.2-beta.1`

**Comment by cyp3rius** [comment]
@kwiat1990 fixed in `v2.2.3`
